docs/docs/icicle/primitives/hash.md

```diff
@@ -71,6 +71,83 @@ The S box power alpha, number of full rounds and partial rounds, rounds constant
 
 In the current version the padding is not supported and should be performed by the user.
 
+## Supported Bindings
+
+[`Rust`](https://github.com/ingonyama-zk/icicle/tree/main/wrappers/rust/icicle-core/src/poseidon2)
```

Comment:
Go also supports Poseidon2, although it is per curve/field, for example for [bn254](https://github.com/ingonyama-zk/icicle/blob/main/wrappers/golang/curves/bn254/poseidon2/poseidon2.go)